pkg/client/unversioned/remotecommand/v3.go

```diff
@@ -108,3 +110,34 @@ func (e *streamProtocolV3) stream(conn streamCreator) error {
 	// waits for errorStream to finish reading with an error or nil
 	return <-errorChan
 }
+
+type decodedExitError struct {
```

Comment:
@ingvagabund I don't see that usage.

@sttts could you alternatively get rid of `decodedExitError` and instead make `remotecommand.ExitError` implement `uexec.ExitError`?